CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 - Stop exporting `actfw_core.__version__`.
 - (Breaking change; for actfw-* developers) Changed the types of `Pipe.in_queues` and `Pipe.out_queues`: `list[Queue[T]]` -> `list[_PadOut[T]]`/`list[_PadIn[T]]`.
 - (Breaking change; for actfw-* developers) Deleted the method `Frame._update()`.
+- (Breaking change; for actfw-* developers) Inheritance of `Task` classes are changed.  For example, `Consumer` and `Producer` were chirldren of `Pipe`, but they are not now.
```

Comment:
Example is not enough for this kind of changes, in my opinion.

Reply:
Right. 5d2bbca